src/proof/dot/dot_printer.h

```diff
@@ -50,9 +50,8 @@ class DotPrinter
    * @param pn the proof node to print
    * @param ruleID the id of the rule to print
    */
-  static void printInternal(std::ostream& out,
-                            const ProofNode* pn,
-                            uint64_t& ruleID);
+  static void printInternal(
+      std::ostream& out, const ProofNode* pn, uint64_t& ruleID, int, bool);
```

Comment:
Add names for the arguments and add documentation above to describe them. Moreove, use `uint64_t` rather than `int`. We favor unsigned variables when we only care for non-negative values. And we specify explicitly the bit-width to avoid possible issues when running in different architectures.